audbackend/core/backend/base.py

```diff
@@ -131,7 +139,9 @@ def close(
             BackendError: if an error is raised on the backend
 
         """
-        utils.call_function_on_backend(self._close)
+        if self.opened:
+            utils.call_function_on_backend(self._close)
+            self.opened = False
```

Comment:
Here I would also recommend we a return value to `self._close()`, which should be `True` on success and then assign `self.opened = not utils.call_function_on_backend(self._close)`.

Reply:
Here, I would leave it symmetrical to `self._open()` and also not introduce a return value.